mod.ts

```diff
@@ -1,5 +1,5 @@
 import  _JSZip from "https://dev.jspm.io/jszip@3.4.0";
-import { WalkOptions, walk } from "https://deno.land/std@v0.51.0/fs/mod.ts";
+import { WalkOptions, walk } from "https://deno.land/std@v0.54.0/fs/mod.ts";
 import { SEP, join } from "https://deno.land/std@v0.51.0/path/mod.ts";
```

Comment:
Can you update all these to 0.54.0 (and the version used in tests .github/workflows to 1.0.5?)

Reply:
also https://github.com/hayd/deno-zip/blob/2745522c174fb5b16f45b452c4c7a5742f04a188/test.ts#L1-L3